### PR TITLE
Ms.rpcs

### PR DESCRIPTION
**Pull request by mariano54**
- RPC api: /push_tx. Using this RPC, you can spend custom chialisp programs. You need to make a SpendBundle, which includes the puzzle reveal (chialisp), a solution (chialisp) and a signature
- RPC apis to query the mempool
- Simplify signature validation logic by using the infinity element

### CODE REVIEW

File: src/types/spend_bundle.py [hunk 1]
```diff
@@ -20,15 +20,16 @@ class SpendBundle(Streamable):
     """
 
     coin_solutions: List[CoinSolution]
-    aggregated_signature: G2Element
+    aggregated_signature: Optional[G2Element]
```
Comment (richardkiss) [inline src/types/spend_bundle.py:23]: Don't do this. Use the 0 signature instead (adding will not change the final signature).

Comment (richardkiss) [reply src/types/spend_bundle.py:23]: AKA `infinity`. See `debug_spend_bundle` below.

Comment (mariano54) [reply src/types/spend_bundle.py:23]: I considered that, but I did this to save space when sending transactions. The serialization of the infinity element is also 96 bytes. What do you think?

File: src/util/errors.py [hunk 2]
```diff
@@ -131,6 +131,7 @@ class Err(Enum):
     BAD_COINBASE_SIGNATURE = 106
     INITIAL_TRANSACTION_FREEZE = 107
     NO_TRANSACTIONS_WHILE_SYNCING = 108
+    ALREADY_INCLUDING_TRANSACTION = 109
```
Comment (richardkiss) [inline src/util/errors.py:134]: you're using 109 twice

File: src/types/spend_bundle.py (hunk 1 above)
Comment (richardkiss) [reply src/types/spend_bundle.py]: A `SpendBundle` without a signature will be a rare thing indeed.

**Review by hoffmang9 — APPROVED** [review]
AOK

**Pull request merged**